modules/cloud/grafana/cloud/README.md

```diff
@@ -45,16 +50,16 @@ The token must have permissions to read stack information. The setup of these pe
 ### `stack`
 
 ```river
-import.git "grafana_cloud" {
-  repository = "https://github.com/grafana/flow-modules.git"
+module.git "grafana_cloud" {
```

Comment:
`module.git` doesn't exist in alloy. I think you want to keep this example as `import.git`

Reply:
Ah, right you are, I'll fix that